dot/node.go

```diff
@@ -308,7 +308,7 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore, stopFunc func()) (*Node,
 	nodeSrvcs = append(nodeSrvcs, sysSrvc)
 
 	// check if rpc service is enabled
-	if enabled := cfg.RPC.Enabled; enabled {
+	if enabled := (cfg.RPC.Enabled || cfg.RPC.WS); enabled {
```

Comment:
Parenthesis is not required.
Not idiomatic to use parenthesis in if-else in Go.

Reply:
removed!